examples/ota-provider-app/esp32/main/BdxOtaSender.cpp

```diff
@@ -138,8 +146,11 @@ void BdxOtaSender::HandleTransferSessionOutput(TransferSession::OutputEvent & ev
         blockData.Data = blockBuf->Start();
         mNumBytesSent  = static_cast<uint32_t>(mNumBytesSent + blockData.Length);
 
-        VerifyOrReturn(CHIP_NO_ERROR == mTransfer.PrepareBlock(blockData),
-                       ChipLogError(BDX, "%s: PrepareBlock failed: %s", __FUNCTION__, chip::ErrorStr(err)));
+        if (CHIP_NO_ERROR != mTransfer.PrepareBlock(blockData))
+        {
+            ChipLogError(BDX, "%s: PrepareBlock failed: %s", __FUNCTION__, chip::ErrorStr(err));
+            mTransfer.AbortTransfer(StatusCode::kUnknown);
```

Comment:
This is not done in the common BDX sender. How did you reproduce to get an error on calling `mTransfer.PrepareBlock`?

Reply:
Oh nevermind, I see you addressed it here: https://github.com/project-chip/connectedhomeip/pull/16615